api/proto/teleport/legacy/client/proto/authservice.proto

```diff
@@ -1814,6 +1814,34 @@ message PaginatedResource {
   reserved "KubeService";
 }
 
+// ResourceWithLogins represents a single resource and associated logins that
+// is allowed to access the resource.
+message ResourceWithLogins {
+  repeated string Logins = 1;
```

Comment:
Why can't we just add this to `PaginatedResource`, why do we need a new type?

I'm also imagining we'll eventually want some other resource metadata like kube groups or db tables, if we're making a new type like this it might be nice to use a more general name than just "logins"